packages/mobile/src/hooks/chat/useConversationsWithRecentMessage.ts

```diff
@@ -118,3 +120,33 @@ const useLastUpdatedChatMessages = () => {
     lastUpdate,
   };
 };
+
+const useLastUpdatedConversations = () => {
+  const queryClient = useQueryClient();
+  const [lastUpdate, setLastUpdate] = useState<number | null>(null);
+
+  useFocusEffect(() => {
+    const lastUpdates = queryClient
```

Comment:
It's a small thing as this would work just as well, but the conversations cache is a single use cache key; There's only a single cache of conversations so would be good to just get the single one instead of reducing a collection of one;

But even then, not wrong to keep it as this, the only thing to change at the very least would be the `exact: false` to `exact: true`